README.md

```diff
@@ -102,6 +102,10 @@ See [`_examples`](_examples) directory for complete standalone examples.
 
     Testing a server running under the [Google App Engine](https://en.wikipedia.org/wiki/Google_App_Engine).
 
+* [`gin_test.go`](_examples/gin_test.go)
+
+    Testing a server utilizing the [gin web framework](https://github.com/gin-gonic/gin).
```

Comment:
Nitpick: I suggest to move it upper, right after the echo example, so that framework examples will be grouped.

Reply:
Resolved. Have moved under echo